stacks/spica/src/function/pages/log-view/log-view.component.html

```diff
@@ -3,10 +3,17 @@ <h4 class="mat-h4">
     <mat-icon>bug_report</mat-icon>
     <span>Logs</span>
   </h4>
-  <button mat-button canInteract="function:logs:delete" (click)="clearLogs()">
-    <mat-icon>clear_all</mat-icon>
-    Clear
-  </button>
+  <div [matTooltip]="'You should choose a function filter to clear logs'">
+    <button
+      mat-button
+      canInteract="function:logs:delete"
+      (click)="clearLogs()"
+      [disabled]="!(queryParams | async)?.function.length"
+    >
+      <mat-icon>clear_all</mat-icon>
+      Clear
+    </button>
+  </div>
```

Comment:
Why we had to wrap this button with a div? If this had been done to use a tooltip, you can use it directly on the button.

Reply:
Because tooltip is disabled when button is disabled. Tooltip should work even if the button is disabled.